db/patterns/vi.eno

```diff
@@ -1,6 +1,6 @@
 name: Vi
 category: advertising
-website_url: http://www.vi.ru/
+website_url: https://www.vi.ru/
```

Comment:
Looks like their website is dead now. I split out a separate issue for that (https://github.com/ghostery/trackerdb/issues/118)

Maybe best to leave it unchanged in this PR (unless you know the new homepage if there is any?).

Reply:
okay, got it.